connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/TestPlugins.java

```diff
@@ -141,18 +179,37 @@ public static void assertAvailable() throws AssertionError {
      * @return A list of plugin jar filenames
      */
     public static List<String> pluginPath() {
-        return PLUGIN_JARS.values()
+        return PLUGIN_JARS.entrySet()
             .stream()
-            .map(File::getPath)
+            .filter(e -> !e.getKey().hideFromPluginPath())
+            .map(e -> e.getValue().getPath())
             .collect(Collectors.toList());
     }
 
+    /**
+     * Gets the path to a single jar containing the test plugin.
+     * @param testPlugin the unique {@link TestPlugin} enum representing the plugin
+     * @return a string representing the plugin jar filename, or null if a plugin for the
```

Comment:
Any reason to return null instead of throwing an exception of some kind if the plugin cannot be found? It looks like the one place where we're currently using this method will fail with an NPE if it does end up returning null, and I can't think of any good cases where we'd want `null` here instead of an exception.